src/CalcViewModel/GraphingCalculator/GraphingSettingsViewModel.h

```diff
@@ -0,0 +1,291 @@
+                        auto number = std::stod(value->Data(), &sz);
+                        if (value->Length() == sz)
+                        {
+                            m_Graph->YAxisMin = m_YMinValue = number;
+                            YMinError = false;
+                        }
+                        else
+                        {
+                            YMinError = true;
+                        }
+                    }
+                    catch (...)
+                    {
+                        YMinError = true;
+                    }
+                }
+                RaisePropertyChanged("YError");
+                RaisePropertyChanged("YMin");
+                UpdateDisplayRange();
+            }
+        }
+
+        property Platform::String ^  YMax
+        {
+            Platform::String ^ get()
+            {
+                return m_YMax;
+            }
+            void set(Platform::String ^  value)
+            {
+                if (m_YMax == value)
+                {
+                    return;
+                }
+                m_YMax = value;
+                if (m_Graph != nullptr)
+                {
+                    try
+                    {
+                        size_t sz;
+                        auto number = std::stod(value->Data(), &sz);
+                        if (value->Length() == sz)
+                        {
+                            m_Graph->YAxisMax = m_YMaxValue = number;
+                            YMaxError = false;
+                        }
+                        else
+                        {
+                            YMaxError = true;
+                        }
+                    }
+                    catch (...)
+                    {
+                        YMaxError = true;
+                    }
+                }
+                RaisePropertyChanged("YError");
+                RaisePropertyChanged("YMax");
+                UpdateDisplayRange();
+            }
+        }
+
+        property int TrigUnit
+        {
+            int get()
+            {
+                return m_Graph == nullptr ? (int)Graphing::EvalTrigUnitMode::Invalid : m_Graph->TrigUnitMode;
+            }
+            void set(int value)
+            {
+                if (m_Graph == nullptr)
+                {
+                    return;
+                }
+                m_Graph->TrigUnitMode = value;
+                RaisePropertyChanged(L"TrigUnit");
+            }
+        }
+
+        property bool TrigModeRadians
+        {
+            bool get()
+            {
+                return m_Graph != nullptr && m_Graph->TrigUnitMode == (int)Graphing::EvalTrigUnitMode::Radians;
+            }
+            void set(bool value)
+            {
+                if (value && m_Graph != nullptr && m_Graph->TrigUnitMode != (int)Graphing::EvalTrigUnitMode::Radians)
+                {
+                    m_Graph->TrigUnitMode = (int)Graphing::EvalTrigUnitMode::Radians;
```

Comment:
Changing the trig mode appears to change the ranges on the Graph but they are not immediately updated in the flyout

Reply:
this should not happen, we call `m_renderer->SetDisplayRanges` with the same value after the user selects a different unit. Fixing SetDisplayRanges will also fix this issue.

Reply:
Can you send me screenshots? From what I see, it doesn't. The issue I see isn't the range but how the graph displays them:
Radian:
![image](https://user-images.githubusercontent.com/1226538/71305486-da689480-2389-11ea-8ca9-6cac4b3dae28.png)

Degree:
![image](https://user-images.githubusercontent.com/1226538/71305488-ece2ce00-2389-11ea-96f1-3df567831e94.png)

Both display the same ranges but the way the graph shows them isn't deterministic